docs/architecture/README.md

```diff
@@ -0,0 +1,17 @@
+# Dashboard Architecture Overview
```

Comment:
I would propose we remove this part from the documentation as it's missing some descriptive text and create a separate for it so that it does not block this PR from being merged

Reply:
I'll have a discussion with @holgerkoser tomorrow about this. Let's decide afterwards.